packages/string-templates/src/index.cjs

```diff
@@ -36,3 +36,8 @@ if (!process.env.NO_JS) {
     return vm.run(js)
   })
 }
+
+const errors = require("./errors")
```

Comment:
Do we still need the `index.cjs`, `index.mjs` and `index.js`? Right now all usage of the library comes through the `index.mjs` - its probably worth clearing this down to just a single endpoint - although likely worth doing in a separate PR.

Reply:
Not sure what is using these indexes. Probably we can clean it, but as you said I will research it in a separated PR :)